File: Array/linearSearch.cpp
#include <iostream>
using namespace std;

var changeArr(int arr[], int size){
    cout << "in function\n";
    for(int i = 0; i < size;i++){
        arr[i]=2*arr[i];
    }
}
int linearSearch(int arr[], int sz, int target){
    for(int i = 0; i < sz; i++){
        if(arr[i] == target){
            return i;
        }
    }
    return -1;
}
int main(){
    int arr[]={4,2,7,8,1,2,5};
    int sz=7;
    int target=50;

    cout << linearSearch(arr, sz, target) << endl;
    return 0;
}

File: Array/linearSearch.js
const isExist = (arr,target)=>{
    for(let i=0;i<arr.length;i++){
        if(arr[i] === target)
            return i;
    }
    return -1;
}
let arr = [10,20,30,40,50,60,70,80,90,100];
let target = 110;

let result = isExist(arr,target);
console.log(result);
